base/file.jl

```diff
@@ -48,10 +46,12 @@ julia> pwd()
 ```
 """
 function pwd()
-    b = Vector{UInt8}(undef, 1024)
-    len = RefValue{Csize_t}(length(b))
-    uv_error(:getcwd, ccall(:uv_cwd, Cint, (Ptr{UInt8}, Ptr{Csize_t}), b, len))
-    String(b[1:len[]])
+    path_max = 1024
+    buf = Base.StringVector(path_max)
```

Comment:
The Base prefix isn't required inside of Base
```suggestion
    buf = StringVector(path_max)
```

Reply:
This file is apart of the Filesystems module